src/components/SMI-Table/index.js

```diff
@@ -4,7 +4,7 @@ import { FaCheckCircle } from "react-icons/fa";
 import { RiCloseCircleFill } from "react-icons/ri";
 import { IoMdHelpCircle } from "react-icons/io";
 import { IconContext } from "react-icons";
-import { TableWrapper } from "./LandscapeTable.style";
+import { TableWrapper } from "./SMITable.style";
```

Comment:
Can't the same stylesheet be used here as well?

Reply:
nope. Have made some updates in the SMI one. Having alternative color in SMI table rows wont make sense since the no. of rows is not constant.

Reply:
Why?
Like for other tables also we don't have constant rows.

Reply:
Infact I don't get the reason for creating a seperate file for the table as well. Even that can be reused.

Reply:
@dhruv0000 we're in need of a multi-dimensional table. Are you thinking of adding a dropdown to choose between SMI versions?